includes/wp-cli.php

```diff
@@ -98,32 +122,66 @@ function import_from_meta( $args, $assoc_args ) {
 		$end_offset = isset( $assoc_args['end'] ) ? intval( $assoc_args['end'] ) : 99999999;;
 		$meta_key = isset( $assoc_args['meta_key'] ) ? sanitize_key( $assoc_args['meta_key'] ) : '';
 		$skip_dupes = isset( $assoc_args['skip_dupes'] ) ? (bool)intval( $assoc_args['skip_dupes'] ) : false;
+		$format = \WP_CLI\Utils\get_flag_value( $assoc_args, 'format' );
 		$dry_run = isset( $assoc_args['dry_run'] ) ? true : false;
+		$verbose = isset( $assoc_args['verbose'] ) ? true : false;
+		$notices = array();
 
 		if ( true === $dry_run ) {
 			WP_CLI::line( "---Dry Run---" );
 		} else {
 			WP_CLI::line( "---Live Run--" );
 		}
 
+		$total_redirects = $wpdb->get_var(
+			$wpdb->prepare(
+				"SELECT COUNT( post_id ) FROM $wpdb->postmeta WHERE meta_key = %s",
+				$meta_key
+			)
+		);
+
+		if ( 0 === absint( $total_redirects ) ) {
+			WP_CLI::error( "No redirects found for that meta_key." );
+		}
+
+		$progress = \WP_CLI\Utils\make_progress_bar( sprintf( 'Importing %s redirects', $total_redirects ), $total_redirects );
```

Comment:
Minor: We should use `number_format` for `$total_redirects` inside the `sprintf`. That way, really large number are easier to read.